### PR TITLE
Add gstatus binary

### PR DESCRIPTION
**Pull request by ChristineTChen**
### What does this PR do?

This PR includes the gstatus command line tool that will be used by the GlusterFS integration.

### Motivation

What inspired you to submit this pull request?

### Additional Notes

Anything else we should know when reviewing?

### Describe your test plan

Write there any instructions and details you may have to test your PR.

### CODE REVIEW

File: release.json [hunk 1]
```diff
@@ -1,4 +1,16 @@
 {
+    "7.23.0-gstatus-build": {
```
Comment (hush-hush) [inline release.json:2]: Should we remove this release before merging to master ?

File: omnibus/config/software/gstatus.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
+#
```
Comment (hush-hush) [inline omnibus/config/software/gstatus.rb:1]: Why here and not in omnibus-software repo ?

File: omnibus/config/software/datadog-agent-integrations-py2.rb [hunk 3]
```diff
@@ -41,6 +41,8 @@
   dependency 'nfsiostat'
   # add libkrb5 for all integrations supporting kerberos auth with `requests-kerberos`
   dependency 'libkrb5'
+  # need for glusterfs
```
Comment (hush-hush) [inline omnibus/config/software/datadog-agent-integrations-py2.rb:44]:
```suggestion
  # needed for glusterfs
```
no ?

**Review by hush-hush — CHANGES_REQUESTED** [review]

File: omnibus/config/software/gstatus.rb (hunk 2 above)
Comment (ChristineTChen) [reply omnibus/config/software/gstatus.rb:1]: I don't have a preference, the last time I added a dep it ended up being here 😅 . I can move this to omnibus-software repo if preferred!

Comment (hush-hush) [reply omnibus/config/software/gstatus.rb:1]: You're right here is fine.

File: omnibus/config/software/datadog-agent-integrations-py3.rb [hunk 4]
```diff
@@ -35,6 +35,8 @@
   dependency 'nfsiostat'
   # add libkrb5 for all integrations supporting kerberos auth with `requests-kerberos`
   dependency 'libkrb5'
+  # need for glusterfs
```
Comment (hush-hush) [inline omnibus/config/software/datadog-agent-integrations-py3.rb:38]: `needed` here too.

**Review by hush-hush — APPROVED** [review]
Added a last comment, fell free to merged after than

**Review by hush-hush — APPROVED** [review]

**Review by mx-psi — APPROVED** [review]

**Pull request merged**